packages/unenv-preset/src/runtime/node/http.ts

```diff
@@ -1,26 +1,21 @@
-// TODO: use the workerd server implementation when available
-// See https://github.com/cloudflare/workerd/pull/4591
 import {
 	_connectionListener,
 	CloseEvent,
-	createServer,
 	maxHeaderSize,
 	MessageEvent,
-	Server,
-	ServerResponse,
 	setMaxIdleHTTPParsers,
+	createServer as unenvCreateServer,
+	Server as unenvServer,
+	ServerResponse as unenvServerResponse,
```

Comment:
Does this mean that we still pull in these unenv polyfills even if the native flag is turned on?
I don't think that our bundlers will be clever enough to tree shake this away since it is a runtime check.
How big are these? Could we have a build time way to do this?

Reply:
Good question.

Yes that will pull `[notImplemented`](https://github.com/unjs/unenv/blob/f9934544231e360a68f1051debb76608659e5cea/src/runtime/_internal/utils.ts#L35) and [`notImplementedClass`](https://github.com/unjs/unenv/blob/f9934544231e360a68f1051debb76608659e5cea/src/runtime/_internal/utils.ts#L59C17-L59C36) - a total of 10 lines of code that are pulled by other polyfills anyway.

I went the runtime way because I think it's simpler.

The build time solution would require 2 hybrid polyfills `http-no-server.ts` and `http-yes-server.ts`, updating the mechanism to load hybrid modules as the name is currently assumed to be the module name. So more code, more changes for no benefit.